server/service/client_mdm.go

```diff
@@ -372,3 +372,17 @@ func (c *Client) prepareAppleMDMCommand(rawCmd []byte) ([]byte, error) {
 	}
 	return b, nil
 }
+
+func (c *Client) MDMLockHost(hostID uint) error {
+	if err := c.authenticatedRequest(nil, "POST", fmt.Sprintf("/api/latest/fleet/hosts/%d/lock", hostID), nil); err != nil {
```

Comment:
to be able to extract the errors

```suggestion
var responseBody lockHostResponse
	if err := c.authenticatedRequest(nil, "POST", fmt.Sprintf("/api/latest/fleet/hosts/%d/lock", hostID), &responseBody); err != nil {
```